packages/models-library/src/models_library/basic_types.py

```diff
@@ -44,6 +44,12 @@ class UUIDStr(ConstrainedStr):
     regex = re.compile(UUID_RE)
 
 
+# non-empty string identifier e.g. "123" or "name_id1" (avoids "" identifiers)
+class IDStr(ConstrainedStr):
```

Comment:
You could summarise the comment inside the name of the class? Personally I would not mind a longer class name here, like: `NonEmptyIDString`